script/run_generate.sh

```diff
@@ -72,7 +72,7 @@ fi
 mkdir -p build/
 cd build/
 
-"${ALPAKA_CI_CMAKE_EXECUTABLE}" --verbose -G "${ALPAKA_CI_CMAKE_GENERATOR}" ${ALPAKA_CI_CMAKE_GENERATOR_PLATFORM}\
+"${ALPAKA_CI_CMAKE_EXECUTABLE}" --log-level=VERBOSE -G "${ALPAKA_CI_CMAKE_GENERATOR}" ${ALPAKA_CI_CMAKE_GENERATOR_PLATFORM}\
```

Comment:
Does this also fix MacOS issues like in #1279?

Reply:
It should. My unverified suspicion is that the CMake version of the Mac hosts has been updated.